src/test/TEST_ij/elast.saved.lassen

```diff
@@ -28,15 +28,15 @@ Final Relative Residual Norm = 7.092339e-10
 
 # Output file: elast.out.7
 Iterations = 17
-Final Relative Residual Norm = 5.873774e-09
+Final Relative Residual Norm = 3.595815e-09
```

Comment:
Why are these differences so large?  Are you expecting them to be?

Reply:
Hi @rfalgout Which difference is "large"? I found the iteration counts were off by 1-2, and the norms were within the same order of magnitude. Maybe I missed something? Do you mean why there are so many? If so, since I changed the truncation routines, the way it breaks ties and the order of the remaining entries are different. So, I expected a lot of results will be changed. (there will be more in `sstruct`, I am running them).

Reply:
Hi Ruipeng.  If the algorithms are "the same", then differences in the first digit would normally be considered quite "large".  It sounds like you are expecting this, though, so no worries.  I just wanted to check.  Thanks!